src/WebJobs.Extensions.CosmosDBCassandra/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.CosmosDBCassandra
+{
+    public static class Constants
+    {
+        public const string DefaultConnectionStringName = "CosmosDBCassandra";
```

Comment:
This is where you name the default settings, if any is applicable. If you want to have a default Contact Point variable name, or default user variable name, then here is where you define them, and wire them at the `CosmosDBCassandraWebJobsBuilderExtensions`.